src/natcap/invest/sdr/sdr_core.pyx

```diff
@@ -392,7 +399,8 @@ def calculate_sediment_deposition(
         e_prime_path)['nodata'][0]
     cdef int col_index, row_index, win_xsize, win_ysize, xoff, yoff
     cdef int global_col, global_row, flat_index, j, k
-    cdef int seed_col, seed_row
+    cdef int seed_col = 0
+    cdef int seed_row = 0
```

Comment:
Wow, how did this even work before?

Reply:
How do you mean?

Reply:
Because I'd use it directly in a logging message below without it being initialized! It'd get set later but the first time through would have made some weird output that I never noticed.